docs/source/markdown/podman-pull.1.md

```diff
@@ -27,7 +27,8 @@ Images are stored in local image storage.
 ## SOURCE
 
  The SOURCE is the location from which the container images are pulled.
- The Image "SOURCE" uses a "transport":"details" format.
+ The Image "SOURCE" uses a "transport":"details" format.  Only the `docker` (container registry)
```

Comment:
You should probably split what was written for `podman run` and `podman create` out into a separate manpage and have all of these reference that page.